source/features/notifications-navigation.tsx

```diff
@@ -0,0 +1,25 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import observe from '../helpers/selector-observer';
+
+function addNextPreviousBtn(navigationButtons: Element): void {
```

Comment:
Can you review this function? I see at least two issues:

- "Select/closest" is repeated
- nextPreviousBtns is never undefined, so `?.` and `if` are not necessary. You can use `!.` you silence typescript